crates/nu-command/src/filesystem/umv.rs

```diff
@@ -0,0 +1,175 @@
+        //MVPS
+        // -f, --force                  do not prompt before overwriting
+        // -i, --interactive            prompt before overwrite
+        // v, --verbose                explain what is being done
+
+        let interactive = call.has_flag("interactive");
+        let no_clobber = call.has_flag("no-clobber");
+        let progress = call.has_flag("progress");
+        let verbose = call.has_flag("verbose");
+        let overwrite = if no_clobber {
+            uu_mv::OverwriteMode::NoClobber
+        } else if interactive {
+            uu_mv::OverwriteMode::Interactive
+        } else {
+            uu_mv::OverwriteMode::Force
+        };
+
+        let paths: Vec<Spanned<String>> = call.rest(engine_state, stack, 0)?;
+        let paths: Vec<Spanned<String>> = paths
+            .into_iter()
+            .map(|p| Spanned {
+                item: nu_utils::strip_ansi_string_unlikely(p.item),
+                span: p.span,
+            })
+            .collect();
+        // CHECK THIS ERROR, DONT KNOW WHAT MOVE GETS
+        if paths.is_empty() {
+            return Err(ShellError::GenericError {
+                error: "Missing file operand".into(),
+                msg: "Missing file operand".into(),
+                span: Some(call.head),
+                help: Some("Please provide source and destination paths".into()),
+                inner: Vec::new(),
+            });
+        }
+        // CHECK THIS ERROR AS WELL
+
+        if paths.len() == 1 {
+            return Err(ShellError::GenericError {
+                error: "Missing destination path".into(),
+                msg: format!("Missing destination path operand after {}", paths[0].item),
+                span: Some(paths[0].span),
+                help: None,
+                inner: Vec::new(),
+            });
+        }
+
+        // Do not glob target
+        let sources = &paths[..paths.len() - 1];
+        let cwd = current_dir(engine_state, stack)?;
+        let mut files: Vec<PathBuf> = Vec::new();
+        for p in sources {
+            let exp_files = arg_glob(p, &cwd)?.collect::<Vec<GlobResult>>();
+            if exp_files.is_empty() {
+                return Err(ShellError::FileNotFound { span: p.span });
+            };
+            let mut app_vals: Vec<PathBuf> = Vec::new();
+            for v in exp_files {
+                match v {
+                    Ok(path) => {
+                        app_vals.push(path);
+                    }
+                    Err(e) => {
+                        return Err(ShellError::ErrorExpandingGlob {
+                            msg: format!("error {} in path {}", e.error(), e.path().display()),
+                            span: p.span,
+                        });
+                    }
+                }
+            }
+            files.append(&mut app_vals);
+        }
+        // Add back the target after globbing
+        let target = paths.last().expect("Should not be reached");
```

Comment:
Could you refactor the `expect()` into some error? We try to minimize potential panics as much as possible in Nushell.